### DISCUSSION TITLE
Standalone usage of file writer

### DISCUSSION THREAD

**Discussion by nnivruth** [moov-io/ach#1173] [General]
I am new to Go and looking to use the ach package as a [Go library](https://github.com/moov-io/ach#go-library). I am trying to create a new project, import the module and run [example_ackWrite_test.go](https://github.com/nnivruth/awesomeProject/blob/main/example_ackWrite_test.go) and it doesn't work. I tried to clone the project and run the example from within the project and that works. 

What is the correct way to use this library?

**Comment by adamdecaf** [comment]
What error(s) are you seeing? Those examples are meant to be inspiration for code in your project.
